Sources/DataPipeline/CIODataPipelineConfig.swift

```diff
@@ -0,0 +1,12 @@
+import CioInternalCommon
+import Foundation
+import Segment
+
+extension Configuration {
+    static var defaultConfiguration: Configuration {
```

Comment:
This is dependent on the `DIGraph` work, so we can get either the `write key` or `site id or API key` to have the config initialized as default.

Reply:
Good call-out. 

Could you add a `TODO` or `FIXME` comment on this line stating that this file is going to change after some other PR is created?